### ISSUE TITLE
Public Folder security check in production mode tests

### ISSUE THREAD

**Issue by ayumi-cloud** [ayumi-cloud/oc-security#3]
> Check to see if they're using a public folder too

Quick question, are you talking about the public_html folder in the cPanel/WHM?We are just talking about your suggestion and thinking of adding a selection option in the settings to choose between a shared hosting and dedicated hosting etc. setup and only showing the public folder check test to non-shared hosting setups.Let me know if this is what you are suggesting and we will code it in later today ... thanks (edited) 
LukeTowers  5 hours ago

No, I'm talking about having the public webroot only have symlinks to the resource folders under a whitelist approach. Shared hosting can have a "public folder" too, what you do then is put the october installation in the home folder and delete the public_html folder and run `php artisan october:mirror public_html --relative`

so basically detect if there is a public or public_html folder in the app root and if there isn't provide information on how / why it should be set up that way.
